examples/cr-console.operator.config.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: operator.openshift.io/v1
+kind: Console
+metadata:
+  name: cluster
+spec:
+  managementState: Managed
+  authentication:
```

Comment:
Same here, @zherman0, `authentication` belongs on the other config, should remove from this example file.